takari-builder-common/src/main/java/io/takari/builder/internal/pathmatcher/PathNormalizer.java

```diff
@@ -37,53 +49,103 @@ public String getBasedir() {
    * {@link PathMatcher}.
    */
   public String normalize(String file) {
-    file = Paths.get(file).normalize().toString();
-    if (file.startsWith(basedir) || basepath.equals(file)) {
+    if (isBasedirOrNestedFile(file)) {
       // avoid expensive normalize0 call if file starts with basedir prefix.
       // this is possible because basedir is normalized in the constructor and PatchMatcher
       // resolves ../ and ./ special directory names.
-      return file;
+      return FIXFS ? fixfs(file) : file;
     }
     return normalize0(file);
   }
 
+  private static String fixfs(String file) {
+    return SEPARATOR_CHAR + file.replace(File.separatorChar, SEPARATOR_CHAR);
```

Comment:
Why is the leading SEPARATOR_CHAR used here?

Reply:
Windows paths look like C:\some\directory but PathMatcher and likely other places expect "absolute" paths that start with '/'

Reply:
I see. so, a normalized windows path would look like this?: /C:/some/directory

Reply:
Right.